internal/cmd/offering/marketplace_test.go

```diff
@@ -143,6 +143,7 @@ var _ = Describe("Marketplace command test", func() {
 		})
 	})
 
+
```

Comment:
shouldn't there be a test here? one that verifies that both service offerings and plans are loaded with the env parameter?